/**:
  ros__parameters:
    transition_timeout: 10.0
    frequency_hz: 10.0
    engage_acceptable_limits:
      allow_autonomous_in_stopped: true  # no check if the velocity is zero, always allowed.
      dist_threshold: 1.5
      yaw_threshold: 0.524
      speed_upper_threshold: 10.0
      speed_lower_threshold: -10.0
      acc_threshold: 1.5
      lateral_acc_threshold: 1.0
      lateral_acc_diff_threshold: 0.5
    stable_check:
      duration: 0.1
      dist_threshold: 1.5
      speed_upper_threshold: 2.0
      speed_lower_threshold: -2.0
      yaw_threshold: 0.262
